src/assignments/phat/lesson_2_answers.clj

```diff
@@ -1,3 +1,38 @@
 (ns lesson-2-answers)
 
-;;; Enter your answers in here.
+;;;;;;;;;;;;;;;;;
+;; first approach
+
+;; split each string into characters => list char-array
+;; sort list and compare
+;; if two lists equal than add the item to anagram vector
+(defn is_anagram
```

Comment:
In clojure, we tend not to use underscores to separate tokens, we use hypens.

See all the built-in functions in the clojure language.